pallets/stream-payment/src/lib.rs

```diff
@@ -425,11 +455,20 @@ pub mod pallet {
             // Unfreeze funds left in the stream.
             T::Assets::decrease_deposit(&stream.config.asset_id, &stream.source, stream.deposit)?;
 
+            let hold_amount = StreamsHolds::<T>::take(stream_id);
```

Comment:
what happens if the hold does not exist?

Reply:
Yes, it returns 0. Sounds good